docs-chef-io/content/inspec/plugins.md

```diff
@@ -60,8 +60,7 @@ inspec plugin install inspec-some-plugin
 inspec plugin install train-some-plugin
 ```
 
-For more details on what the `plugin` command can do, see the [online help](/inspec/cli/#plugin),
-or run `inspec plugin help`.
+For more details on what the `plugin` command can do, run `inspec plugin help`.
```

Comment:
This linked to [an existing document](https://docs.chef.io/inspec/cli/), but that does not contain any references to the `inspec plugin` command so I removed the link.

Reply:
This sentence will probably need to be re-worded then.

Reply:
As a follow-up to this PR we should keep an issue open for the fact that https://docs.chef.io/inspec/cli/ does not mention the `plugin` or `init` commands, I'll raise that now.

Reply:
https://github.com/inspec/inspec/issues/5291

Reply:
@phiggins Scratch that about more rewording. I had more coffee. 😅 Your rewording should be good as is.
@james-stocks Thanks in advance for filing that issue!